scripts/release-eval-smoke-sitecustomize.py

```
@@ -0,0 +1,179 @@
+        kwargs = self.config.agent.kwargs
+        reader, writer = await asyncio.open_connection(kwargs.relay_host, kwargs.relay_port)
+
+        def send(value):
+            writer.write((json.dumps(value, separators=(",", ":")) + "\n").encode())
+
+        send(
+            {
+                "token": kwargs.relay_token,
+                "kind": "ready",
+                "instruction": "release smoke",
+                "cwd": "/tmp",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)